code/processes/centraltailsort.q

```diff
@@ -1,40 +1,36 @@
-.proc.loadf [getenv[`KDBCODE],"/wdb/common.q"]                             /-load common wdb parameters & functions
 \d .ts
 
+.servers.CONNECTIONS:(distinct .servers.CONNECTIONS,.servers.tailsorttypes,.servers.hdbtypes,.servers.rdbtypes);
+.servers.startup[];
+
 taildir:hsym `$getenv`KDBTAIL;                                             /-load in taildir env variables
 hdbdir:hsym `$getenv`KDBHDB;                                               /-load in hdb env variables
-rdbtypes:@[value;`rdbtypes;`rdb];                                          /- rdbs to send reset window message to
 taildbs:key taildir;                                                       /-list of tailDBs that need saved to HDB
-taildirs:();                                                               /-empty list to append tailDB paths to - to be used
-savelist:@[value;`savelist;`quote`trade];                                  /-list of tables to save to HDB
+/get all tables from the schemalist
+schematables:first each first select schemalist from [first exec w from .servers.SERVERS where proctype=.wdb.tickerplanttypes](`subdetails;`;`); 
+savelist:.ts.schematables except .wdb.ignorelist;                          /-list of tables to save to HDB
 reloadorder:@[value;`reloadorder;`hdb`rdb];
 eodwaittime:@[value;`eodwaittime;0D00:00:10.000];                          /-length of time to wait for async callbacks to complete at eod
-
-.servers.CONNECTIONS:(distinct .servers.CONNECTIONS,.servers.tailsorttypes,.servers.hdbtypes,.servers.rdbtypes);
-.servers.startup[];
+date:.z.d;								   /-date variable to ensure correct eod
```

Comment:
The only thing this is used for is to set the date on start up so it can be incremented. By the time the eod stuff happens in the centraltailsort it's just after midnight so we use the date got from start up and increment it as the last part of the eod. I think .z.d is fine but I haven't heard of .proc.cd[], do yo think I should look into using it?

Reply:
tbh I'm not sure - all `.proc.cd[]` does is return either `.z.d` or `.z.D` depending on if `-localtime` param is set (I think there's a column in `process.csv` to set this).

I think maybe it should use it to be safe, it's literally just 

```suggestion
date:.proc.cd[];								   /-date variable to ensure correct eod
```